events/php/event_feed.php

```diff
@@ -242,22 +242,24 @@ function inspect_event_page($xml, $categories){
 function display_on_feed_events($page_info){
     global $StartDate;
     global $EndDate;
-    $modifiedStartDate = $StartDate / 1000;
-    $modifiedEndDate = $EndDate / 1000;
 
     //Check if the event falls between the given range.
     if( $StartDate != "" && $EndDate != "" ){
+        $modifiedStartDate = $StartDate / 1000;
```

Comment:
Why did you move these variables? Is it more efficient to have them inside the if statements?

Reply:
I would have preferred them to be on the outside, but this was to fix the php7 warnings. Instead of trying to use $EndDate when it doesn't exist, we check to see if it exists first. So with that in mind, I just put them into the if statements